### PR TITLE
Use Log instead of Logs

### PR DESCRIPTION
**Pull request by reynir**
I noticed my application was logging happy-eyeballs warnings. This was slightly confusing to me.

### CODE REVIEW

**Comment by reynir** [comment]
This gets logged *a lot* for me in opam-mirror. I'm not sure what it means or why it should be a warning...

**Comment by hannesm** [comment]
Are doing a lot of concurrent requests? The log message appears if there's already a task querying for AAAA records (IPv6 address records) for a given hostname whose A record (IPv4) already exists -- but we give IPv6 a bit head start (since we prefer IPv6 over IPv4).

We can demote to Log.info or Log.debug if that makes your life / output easier :)

**Comment by reynir** [comment]
Yes, I think it defaults to up to 20 concurrent downloads, and a lot of the archives are on the same few hosts (e.g. github)

File: src/happy_eyeballs.ml [hunk 1]
```diff
@@ -329,9 +329,9 @@ let event t now e =
                   Connect (name, id, c.attempt, dst) :: actions
                 | Resolving _ts -> Waiting_for_aaaa (now, ips), c.attempt, actions
                 | Waiting_for_aaaa (ts, ips') ->
-                  Logs.warn (fun m -> m "already waiting for AAAA with %a"
-                                Fmt.(list ~sep:(any ", ") Ipaddr.V4.pp)
-                                (Ipaddr.V4.Set.elements ips'));
+                  Log.warn (fun m -> m "already waiting for AAAA with %a"
```
Comment (hannesm) [inline src/happy_eyeballs.ml:332]:
```suggestion
                  Log.debug (fun m -> m "%a already waiting for AAAA with %a"
```

File: src/happy_eyeballs.ml [hunk 2]
```diff
@@ -329,9 +329,9 @@ let event t now e =
...
+                               Fmt.(list ~sep:(any ", ") Ipaddr.V4.pp)
```
Comment (hannesm) [inline src/happy_eyeballs.ml:333]:
```suggestion
                               Domain_name.pp name
                               Fmt.(list ~sep:(any ", ") Ipaddr.V4.pp)
```

**Comment by hannesm** [comment]
should we do a release?

**Comment by reynir** [comment]
I think that would be nice

**Pull request merged**